pkg/providers/docker/workers_test.go

```diff
@@ -131,6 +131,69 @@ func TestWorkersSpecUpgradeCluster(t *testing.T) {
 	g.Expect(workers.Groups).To(ConsistOf(*expectedGroup1, *expectedGroup2))
 }
 
+func TestWorkersSpecUpgradeClusterRemoveLabels(t *testing.T) {
```

Comment:
can we move this method to `clusterapi/workers_test.go`?

Reply:
I already added a test case in `clusterapi/workers_test.go` to make sure that the kubeadmConfigTemplate name changes if the labels are removed. Do you think there should be another test in that file?